.github/workflows/ci.yaml

```diff
@@ -251,6 +251,12 @@ jobs:
           cache-from: type=gha
           cache-to: type=gha,mode=max
           build-args: MIX_ENV=${{ env.MIX_ENV }}
+      - name: Remotely trigger trento-web demo deployment
```

Comment:
We could maybe add an if condition to deploy only when we want:
```
if: vars.DEPLOY_DEMO == 'true'
```
This way we continue having control over deploying or not.